mm/kfence.c

```diff
@@ -271,6 +272,7 @@ void *guarded_alloc(size_t size, gfp_t gfp)
 	void *obj = NULL, *ret;
 	struct kfence_freelist_t *item;
 	int index = -1;
+	bool right = get_random_int() % 2;
```

Comment:
I think prandom_u32_max is simpler. I don't think we need the guarantees that get_random provides.

Reply:
Fair enough. Done.